src/containers/BookForm.js

```diff
@@ -27,6 +27,12 @@ class BookForm extends Component {
     const { submitNewBook } = this.props;
     submitNewBook(this.state);
     e.target.reset();
+
+    this.setState({
```

Comment:
- Very nice addition resetting the form values after a submit event, I see you thought about the UX for the form. 🌟